operatorconfig/driverconfig/powerflex/v2.2.0/controller.yaml

```diff
@@ -0,0 +1,234 @@
+          imagePullPolicy: IfNotPresent
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--v=5"
+            - "--leader-election=true"
+            - "--enable-node-watcher=true"
+            - "--http-endpoint=:8080"
+            - "--monitor-interval=60s"
+            - "--timeout=180s"
+          env:
+            - name: ADDRESS
+              value: /var/run/csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/run/csi
+        - name: vg-snapshotter
+          image: dellemc/csi-volumegroup-snapshotter:v1.0.0
+          imagePullPolicy: IfNotPresent 
+          env:
+            - name: ADDRESS
+              value: /var/run/csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/run/csi
+        - name: snapshotter
+          image: k8s.gcr.io/sig-storage/csi-snapshotter:v5.0.1
+          imagePullPolicy: IfNotPresent
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--timeout=120s"
+            - "--v=5"
+            - "--leader-election=true"
+          env:
+            - name: ADDRESS
+              value: /var/run/csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/run/csi
+        - name: resizer
+          image: k8s.gcr.io/sig-storage/csi-resizer:v1.4.0
+          imagePullPolicy: IfNotPresent
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--v=5"
+            - "--leader-election=true"
+          env:
+            - name: ADDRESS
+              value: /var/run/csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/run/csi
+        - name: driver
+          image: dellemc/csi-vxflexos:v2.2.0
+          imagePullPolicy: IfNotPresent
+          command: [ "/csi-vxflexos.sh" ]
+          args:
+            - "--leader-election"
+            - "--array-config=/vxflexos-config/config"
+            - "--driver-config-params=/vxflexos-config-params/driver-config-params.yaml"
+          env:
+            - name: CSI_ENDPOINT
+              value: /var/run/csi/csi.sock
+            - name: X_CSI_MODE
+              value: controller
+            - name: X_CSI_VXFLEXOS_ENABLESNAPSHOTCGDELETE
+              value: false
+            - name: X_CSI_VXFLEXOS_ENABLELISTVOLUMESNAPSHOT
+              value: false
+            - name: SSL_CERT_DIR
+              value: /certs
+            - name: X_CSI_HEALTH_MONITOR_ENABLED
+              value: false
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/run/csi
+            - name: vxflexos-config
+              mountPath: /vxflexos-config
+            - name: vxflexos-config-params
+              mountPath: /vxflexos-config-params
+      volumes:
+        - name: socket-dir
+          emptyDir:
+        - name: vxflexos-config
+          secret:
+            secretName: vxflexos-config
```

Comment:
we should use DriverDefaultReleaseName to search for things like secret/config map in csm-operator.

This will keep it consistent with what was done w/ PScale:
 ```
   - name: isilon-configs
          secret:
            secretName: <DriverDefaultReleaseName>-creds
        - name: csi-isilon-config-params
          configMap:
             name: <DriverDefaultReleaseName>-config-params
```

Reply:
yes 
changing it